wsd/COOLWSD.cpp

```diff
@@ -2061,18 +2062,18 @@ void COOLWSD::innerInitialize(Poco::Util::Application& self)
         { "logging.file.property[0][@name]", "path" },
         { "logging.file.property[1]", "never" },
         { "logging.file.property[1][@name]", "rotation" },
-        { "logging.file.property[2]", "true" },
-        { "logging.file.property[2][@name]", "compress" },
-        { "logging.file.property[3]", "false" },
-        { "logging.file.property[3][@name]", "flush" },
+        { "logging.file.property[2]", "false" },
+        { "logging.file.property[2][@name]", "archive" },
+        { "logging.file.property[3]", "true" },
+        { "logging.file.property[3][@name]", "compress" },
         { "logging.file.property[4]", "10 days" },
         { "logging.file.property[4][@name]", "purgeAge" },
         { "logging.file.property[5]", "10" },
         { "logging.file.property[5][@name]", "purgeCount" },
         { "logging.file.property[6]", "true" },
         { "logging.file.property[6][@name]", "rotateOnOpen" },
         { "logging.file.property[7]", "false" },
-        { "logging.file.property[7][@name]", "archive" },
+        { "logging.file.property[7][@name]", "flush" },
```

Comment:
ah, this pair of hunks just reindexes these into the order they appear in coolwsd.xml. I was confused because the default I see in coolwsd.xml for archive is "timestamp" not "true" of "false", but it was like that anyway.